person.rb

```diff
@@ -29,10 +29,3 @@ def can_use_services?
     of_age? || @parent_permission
   end
 end
```

Comment:
- [ ] @Irirwanirira great work implementing the changes requested however your initialize method is not allowing more than 2 arguments, I would suggest you check the parent_permission argument and change it to allow the user to enter the third argument
![image](https://user-images.githubusercontent.com/61408860/210953834-1c528ec5-9f13-4141-aa3b-e869fdb82a06.png)

Reply:
@danmainah  Thank you for your kind review